diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6fbc1bf..64ab6af 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -226,30 +226,10 @@ if ( WIN32 )
 
   set ( LIBFLUID_CPPFLAGS "-DFLUIDSYNTH_DLL_EXPORTS" )
   set ( FLUID_CPPFLAGS "-DFLUIDSYNTH_NOT_A_DLL" )
-  if  ( MSVC )
-    # statically link in the CRT library to avoid a bunch of runtime DLL dependencies and allow 
-    # the CI windows builds to be run under WinXP
-    foreach ( flag_var
-        CMAKE_C_FLAGS
-        CMAKE_C_FLAGS_DEBUG
-        CMAKE_C_FLAGS_RELEASE
-        CMAKE_C_FLAGS_MINSIZEREL
-        CMAKE_C_FLAGS_RELWITHDEBINFO
-        CMAKE_CXX_FLAGS
-        CMAKE_CXX_FLAGS_DEBUG
-        CMAKE_CXX_FLAGS_RELEASE
-        CMAKE_CXX_FLAGS_MINSIZEREL
-        CMAKE_CXX_FLAGS_RELWITHDEBINFO
-        )
-      if ( ${flag_var} MATCHES "/MD" )
-        string ( REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}" )
-      endif ( ${flag_var} MATCHES "/MD" )
-    endforeach ( flag_var )
-    
-  else ( MSVC )
+  if  ( NOT MSVC )
     # only set debug postfix if not MSVS building
     set ( CMAKE_DEBUG_POSTFIX "_debug" )
-  endif ( MSVC )
+  endif ( NOT MSVC )
   
   # MinGW compiler (a Windows GCC port)
   if ( MINGW )
